### ISSUE TITLE
4x4 cant edit sequence blocks on Mac

### ISSUE THREAD

**Issue by z-plane** [squinkylabs/SquinkyVCV#187]
I can record data into 4x4 sequence blocks but then can't edit it again using seq++. No sequence data appears... the same patch works fine on PC :/

Testing on Mac was running OSX Mojave.

**Comment by squinkylabs** [comment]
Interesting, I'll have to try that. You are remembering to enable remote editing on Seq++ on the mac, yes? 
![image](https://user-images.githubusercontent.com/16314838/102721166-0032bd00-42ae-11eb-8bba-6cf0ef7108b2.png)

**Comment by z-plane** [comment]
yes, the remote option is ticked, and after further testing on PC I have since been unable to recall data stored in a 4x4 sequence block. :/

**Comment by squinkylabs** [comment]
Do you mean you save a patch with data in 4X4 and when you reload it, it isn't there? Or do you mean the remote editing doesn't work on PC either for you?

**Comment by z-plane** [comment]
> Do you mean you save a patch with data in 4X4 and when you reload it, it isn't there?

Yes, this is the issue.
